docker/nodejs/express/Dockerfile

```diff
@@ -5,3 +5,6 @@ COPY package.json .npmrc app.js /app/
 
 WORKDIR /app
 RUN npm install
+
+ARG NODE_AGENT_PACKAGE=
+RUN npm install $NODE_AGENT_PACKAGE
```

Comment:
Given the below command:

```bash
python3 scripts/compose.py \
  start 8.2.0 \
  --nodejs-agent-package elastic/apm-agent-nodejs#112a13c333a3db3ca0c99358fc688c31bb5f6cb7 \
  --opbeans-node-agent-branch 112a13c333a3db3ca0c99358fc688c31bb5f6cb7 \
  --build-parallel   \
  --with-agent-nodejs-express   \
  --with-opbeans-node   \
  --no-apm-server-self-instrument   \
  --apm-server-agent-config-poll=1s   \
  --force-build
```

it produces the below output

<img width="1396" alt="image" src="https://user-images.githubusercontent.com/2871786/169070100-4ee78c32-3ce9-4a66-823a-19cac1e839ce.png">


<img width="1108" alt="image" src="https://user-images.githubusercontent.com/2871786/169070445-d588cd43-f141-4b97-9163-4a5a36d49c5c.png">